raiden/network/transport/matrix/transport.py

```diff
@@ -1192,29 +1198,53 @@ def _send_with_retry(self, queue: MessagesQueue) -> None:
         retrier.enqueue(queue_identifier=queue.queue_identifier, messages=queue.messages)
 
     def _send_raw(self, receiver_address: Address, data: str) -> None:
-        room = self._get_room_for_address(receiver_address, require_online_peer=True)
+        assert self._raiden_service is not None, "_raiden_service not set"
+
+        # Try sending using toDevice message
+        capabilities = self._address_mgr.get_address_capabilities(receiver_address)
+        if self._capability_usable(Capabilities.TODEVICE, capabilities):
+            self.log.debug(
+                "Send raw using toDevice message",
+                receiver=to_checksum_address(receiver_address),
+                data=data.replace("\n", "\\n"),
+            )
+            online_userids = {
+                user_id
+                for user_id in self._address_mgr.get_userids_for_address(receiver_address)
+                if self._address_mgr.get_userid_presence(user_id) in USER_PRESENCE_REACHABLE_STATES
+            }
+            if not online_userids:
+                self.log.debug("No online user_ids", online_userids=online_userids)
+                return
+
+            body = {
+                user_id: {"*": {"msgtype": "m.text", "body": data}} for user_id in online_userids
+            }
+
+            self._client.api.send_to_device(event_type="m.room.message", messages=body)
```

Comment:
sounds a bit odd to use `m.room.message` as an event type. Isn't there anything which is more fitting (since we do not use rooms in this case)?